pkgs/development/python-modules/deepl/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, requests
+, poetry-core
+}:
+
+buildPythonPackage rec {
+  pname = "deepl";
+  version = "1.13.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Rq7L/rgfJJ3ZspuL1IfZv+x60t8cZRPkrVryJf12WLk=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    requests
```

Comment:
Should we add the optional `keyring` dependency too? It seems to work as-is but it might be nice to have for some people

Reply:
good idea, I should've catch it, it's in the poetry dependencies.